index.js

```diff
@@ -19,6 +21,14 @@ Settings.loadRepositoriesSettings('./config.json')
         app.listen(Settings.port, function () {
             dateLog('cesium-concierge listening on port ' + Settings.port);
         });
+
+        // Run every night.
+        schedule.scheduleJob('0 22 * * *', function(){
```

Comment:
formatting

`function() {`

Are you set up to run the auto-formatter?

Reply:
Sorry! Yes, I've got it on VScode but I've been running with Sublime lately just because it's faster. I'll see if there's a plugin for Sublime for that.